### PR TITLE
docs: wind up translations

### PR DESCRIPTION
**Pull request by derbalkon**
- 为了方便翻译收尾，把文档更新到了 Configuration.pdf 释出的 [badcd12](https://github.com/acidanthera/OpenCorePkg/commit/badcd125bdb8043d4bc6d800a0fbf16716b7359d) 版本
- UEFI 的 `AudioCodec`, `AudioDevice`, `AudioOut` 部分，原文档中有对代码的斜体加粗修饰，纯 Markdown 好像没办法搞定，所以嵌入了一些 HTML，用 Doku 看了一下似乎没什么排版上的问题，如果有其他影响的话就撤回好啦
- 完结并向苏卡卡撒花w（暂时

### CODE REVIEW

File: source/12-troubleshooting.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: 12. 排错
 description: 当你遇到问题的时候应该看看这个
 type: docs
 author_info: 由 xMuu、Sukka 整理，由 Sukka 翻译
```
Comment (SukkaW) [inline source/12-troubleshooting.md:5]: Would you mind adding your name to it?

E.g.

```suggestion
author_info: 由 xMuu、Sukka、	derbalkon 整理，由 Sukka、derbalkon 翻译
```

So does the other pages.

Comment (derbalkon) [reply source/12-troubleshooting.md:5]: Right, totally forgot! Will do.

**Comment by derbalkon** [comment]
Hmm? It looks like the change committed to index.md doesn't seem to be deployed on the website.

**Comment by SukkaW** [comment]
@derbalkon Checks again?

It might be caused by CDN cache.

**Comment by derbalkon** [comment]
@SukkaW Caused by CDN cache indeed. All good now. Much obliged!

**Pull request merged**